plugins/protocolbuffers/pyi/v24.4/buf.plugin.yaml

```diff
@@ -7,3 +7,12 @@ output_languages:
   - python
 spdx_license_id: BSD-3-Clause
 license_url: https://github.com/protocolbuffers/protobuf/blob/v24.4/LICENSE
+registry:
+  python:
+    package_type: "stub-only"
+    # https://github.com/protocolbuffers/protobuf/blob/v24.4/python/setup.py#L471
+    requires_python: ">=3.7"
+    deps:
+      # https://pypi.org/project/protobuf/
+      - "protobuf"
+      - "types-protobuf"
```

Comment:
What do you think about just making these changes to start on the latest versions of the plugins?

Reply:
reverted in https://github.com/bufbuild/plugins/pull/899/commits/ef3fb6865918d718e4f8bd72b67f08e0abaa2d1e